facade/src/main/java/org/jboss/pnc/facade/providers/DeliverableAnalyzerReportProviderImpl.java

```diff
@@ -98,6 +126,88 @@ public Page<AnalyzedArtifact> getAnalyzedArtifacts(
                 analyzedArtifacts);
     }
 
+    @RolesAllowed(SYSTEM_USER)
+    @Override
+    public void addLabel(String id, DeliverableAnalyzerReportLabelRequest request) {
```

Comment:
This method must be in one transaction. When you changed the class to ApplicationScoped, this is no longer in transaction.

Reply:
Yess! [37d3a94](https://github.com/project-ncl/pnc/pull/3840/commits/37d3a94f83496ac9f93d61a0ed55fb2115d85b42)